src/commcare_cloud/commands/ansible/run_module.py

```diff
@@ -197,3 +193,30 @@ def run(self, args, unknown_args):
         args.quiet = True
         del args.shell_command
         return RunAnsibleModule(self.parser).run(args, unknown_args)
+
+
+class Ping(CommandBase):
+    command = 'ping'
+    help = 'Ping specified or all machines to see if they have been provisioned yet.'
+
+    arguments = (
+        shared_args.INVENTORY_GROUP_ARG,
+        Argument('--silence-warnings', action='store_true',
+                 help="Silence shell warnings (such as to use another module instead)"),
+    ) + NON_POSITIONAL_ARGUMENTS
+
+    def modify_parser(self):
+        RunAnsibleModule(self.parser).modify_parser()
+
+    def run(self, args, unknown_args):
+        ping_shell_command = 'echo {{ inventory_hostname }}'
+
+        args.module = 'shell'
+        if args.silence_warnings:
+            args.module_args = 'warn=false ' + ping_shell_command
+        else:
+            args.module_args = ping_shell_command
+        args.skip_check = True
+        args.quiet = True
```

Comment:
A bunch of this smacks of being copy-pasted from `RunShellCommand`. Conceptually this command (`ping`) is trying to basically run `run-shell-command 'echo {{ inventory_hostname }}'`, right? So I'd advocate for _calling_ `RunShellCommand.run` rather than _copying_ its contents.

Reply:
I'm not confident in my change in line 207 (because the user could specify something besides the default, which isn't really the point of this), do you have advice on that?
https://github.com/dimagi/commcare-cloud/pull/1702/commits/e41d302be1014e4dacd3f3ad27eaa9dfb12f3aef